integr8ly/cloud-resource-operator
# Intly 6508 remove smtp providers

## Overview

Jira: https://issues.redhat.com/browse/INTLY-6508

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make cluster/seed/workshop/redis`
- Run `make cluster/seed/workshop/postgres`
- Run the operator
- Ensure resources create successfully
